src/library_fs.js

```diff
@@ -1271,7 +1271,7 @@ mergeInto(LibraryManager.library, {
       FS.mkdev('/dev/tty1', FS.makedev(6, 0));
       // setup /dev/[u]random
       var random_device;
-      if (typeof crypto !== 'undefined') {
+      if (!ENVIRONMENT_IS_NODE && typeof crypto !== 'undefined') {
```

Comment:
Probably cleaner to move node branch to the top: `if node ... else if crypto ... else ...`

Reply:
probably yes